_posts/2018-09-25-no65-dear-moon-hacks-slides.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: slides
+title: Berlin Hack & Tell \#65 - Dear Moon Hacks
+date: 2018-09-25
+time: '18:30'
+category: slides
```

Comment:
is it worth flipping the examples to have the 'static' parts (i.e. same in every post) to the top so it's easier to know what needs to change?

Reply:
Yes, great idea!

Reply:
I found an even better solution, Jekyll 2.0 supports front matter defaults, so I've moved these to `_config.yml` and now it's even more clear what you're suppose to change.

https://jekyllrb.com/docs/configuration/front-matter-defaults/